comps/comps-foreman-plugins-el8.xml

```diff
@@ -195,9 +195,11 @@
       <packagereq type="default">rubygem-fast_gettext-doc</packagereq>
       <packagereq type="default">rubygem-fog-kubevirt-doc</packagereq>
       <packagereq type="default">rubygem-fog-proxmox-doc</packagereq>
+      <packagereq type="default">rubygem-foreman-tasks-core-doc</packagereq>
```

Comment:
this file is missing the addition of your new package (and it's -doc subpackage)